javascript/big-practice/src/scripts/components/inputSearchComponent.js

```diff
@@ -13,7 +13,7 @@ const inputSearchComponent = () => {
     <input type="text" name="search" class="search" placeholder="Search" autocomplete="off">
 
     <figure class="icon-close-cover">
-      <img class="icon-close" src="${iconClose}" alt="icon close">
+      <img class="icon-close hide" src="${iconClose}" alt="icon close">
```

Comment:
You should use CSS to style display: none for this class, and when you want to show this icon, you should handle on that property